srv/modules/runners/populate.py

```diff
@@ -569,14 +569,14 @@ def igw_members(self):
         Note: identical to above
         """
         minion_dir = "{}/role-igw/stack/default/{}/minions".format(self.root_dir, self.cluster)
-        add_pub_interface(minion_dir)
+        self.add_pub_interface(minion_dir)
 
     def ganesha_members(self):
         """
         Create a file for ganesha hosts.
         """
         minion_dir = "{}/role-ganesha/stack/default/{}/minions".format(self.root_dir, self.cluster)
-        add_pub_interface(minion_dir)
+        self.add_pub_interface(minion_dir)
 
     def add_pub_interface(self, minion_dir):
```

Comment:
Is it is only used within the class even ```__add_pub_interface```. It won't be truly private but at least the intent is clear.